arch/arm/boot/dts/aspeed/aspeed-bmc-amd-onyx.dts

```diff
@@ -26,3 +26,98 @@
 		reg = <0x4c>;
 	};
 };
+
+#include <dt-bindings/i3c/i3c.h>
+
+#define JESD300_SPD_I2C_MODE(bus, index, addr) \
```

Comment:
As discussed, please modify it in a manner that will be helpful for DT overlay in the future.

Reply:
Temporary change. Subject to change after DT overlay is implemented.